src/components/ComponentDetails/index.tsx

```diff
@@ -1,16 +1,23 @@
 import React from "react";
 import componentJson from "@ukic/docs";
+import canaryComponentJson from "@ukic/canary-docs";
 import EventTable from "./EventTable";
 import MethodTable from "./MethodTable";
 import PropTable from "./PropTable";
 import SlotTable from "./SlotTable";
 import StyleTable from "./StyleTable";
 
 const ComponentDetails: React.FC<{ component: string }> = ({ component }) => {
-  const componentDetails = componentJson.components.find(
+  let componentDetails = componentJson.components.find(
     (currentComponent) => currentComponent.tag === component
   )!;
 
+  // If the component is not found in the main component JSON, check the canary component JSON
+  if (componentDetails === undefined) {
+    componentDetails = canaryComponentJson.components.find(
+      (currentComponent) => currentComponent.tag === component
+    )!;
+  }
```

Comment:
just a thought. could it pass a boolean `canary` prop to the ComponentDetails & then this code would be much simpler eg
```
const ComponentDetails: React.FC<{ component: string, canary=false }> = ({ component }) => {
  const jsonData = canary ? componentJson : canaryComponentJson;
  const componentDetails = jsonData.components.find(
  ...
```

Reply:
Amended using this suggestion